ui/src/components/credentials/CredentialIssuedQR.tsx

```diff
@@ -6,12 +6,14 @@ import { CredentialQR } from "src/components/credentials/CredentialQR";
 import { ErrorResult } from "src/components/shared/ErrorResult";
```

Comment:
I think this component should be removed. Let's wait for [the conclusions of this thread](https://privadoid.slack.com/archives/C074GNHEN82/p1728905716442129).